app/components/card.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="card__reusable">
```

Comment:
we generally add `__` for childs under the block in BEM convention.

It's like,
`block__element--modifier`

example:
`card__childElementOfCard--md`